tests/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN pip install poetry
 
 COPY pyproject.toml poetry.lock ./
 
-RUN poetry export --without-hashes -f requirements.txt > requirements.txt
+RUN poetry export --without-hashes -f requirements.txt --output requirements.txt
```

Comment:
Without this change, we add an irrelevant record to the requirements file which blocks us from building the image.

`Creating virtualenv tests-eJoqSNov-py3.9 in /Users/uriweisman/Library/Caches/pypoetry/virtualenvs`